From a502cd08088343ef2e446cfc8c9c591a7fdacd9e Mon Sep 17 00:00:00 2001
From: Edson Juliano Drosdeck <edson.drosdeck@gmail.com>
Date: Thu, 4 Jul 2024 19:38:38 -0300
Subject: [PATCH] Fix key toggle touchpad button for multilaser ul154 (#33630)

(cherry picked from commit 3faee9928de01ada0f08160cdfa308f044b84700)

Related: RHEL-5950
---
 hwdb.d/60-keyboard.hwdb | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 48aa92b433..607515b7c3 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -2214,6 +2214,13 @@ evdev:name:SIPODEV USB Composite Device:dmi:bvn*:bvr*:svnPositivoTecnologiaSA:pn
  KEYBOARD_KEY_7006d=prog3                               # Programmable button
  KEYBOARD_KEY_7006e=prog4                               # Programmable button
 
+###########################################################
+# Multilaser
+###########################################################
+# Multilaser Ultra (UL154)
+evdev:name:AT Translated Set 2 keyboard:dmi:bvn*bvr*:svnMultilaserIndustrial:pn*:pvr*:rvn*:rnUL154*
+ KEYBOARD_KEY_76=f21                                    # Fn+f2 toggle touchpad
+
 ###########################################################
 # Other
 ###########################################################
